java-compat/src/Ice/src/main/java/IceInternal/IncomingBase.java

```diff
@@ -280,6 +299,14 @@
     {
         assert(_responseHandler != null);
 
+        if(_response)
+        {
+            //
+            // If there's already a response output stream, reset it to re-use it
```

Comment:
This comment is inaccurate since it calls _os.reset() all the time, which means there is already a response output stream.